php/src/AbstractJws.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TrueLayer\Signing;
+
+use Exception;
+use TrueLayer\Signing\Contracts\Jws as IJws;
+use TrueLayer\Signing\Exceptions\RequestPathNotFoundException;
+
+abstract class AbstractJws implements IJws
+{
+    /**
+     * @var string
```

Comment:
I wouldn't change this now, but do we need these type definitions when we have the types defined in the code?

Reply:
I think PHPStan complains if we don't provide them.

Reply:
I don't remember it complaining about them before :thinking: